src/clients/client.ts

```diff
@@ -372,27 +371,19 @@ class Client {
   ///
 
   /**
-   * Triggers a dump creation process
+   * Creates a dump
    * @memberof MeiliSearch
    * @method createDump
-   * @returns {Promise<EnqueuedDump>} Promise returning object of the enqueued task
+   * @returns {Promise<EnqueuedTask>} Promise returning object of the enqueued task
    */
-  async createDump(): Promise<EnqueuedDump> {
+  async createDump(): Promise<EnqueuedTask> {
     const url = `dumps`
-    return await this.httpRequest.post<undefined, EnqueuedDump>(url)
+    return await this.httpRequest.post<undefined, EnqueuedTask>(url)
   }
 
-  /**
-   * Get the status of a dump creation process
-   * @memberof MeiliSearch
-   * @method getDumpStatus
-   * @param {string} dumpUid Dump UID
-   * @returns {Promise<EnqueuedDump>} Promise returning object of the enqueued task
-   */
-  async getDumpStatus(dumpUid: string): Promise<EnqueuedDump> {
-    const url = `dumps/${dumpUid}/status`
-    return await this.httpRequest.get<EnqueuedDump>(url)
-  }
+  ///
+  /// TOKENS
+  ///
```

Comment:
? 🙃

Reply:
I forgot the separation with the tokens section just bellow so I added it 😅